week2/homework/src/help.txt

```diff
@@ -0,0 +1,10 @@
+
+===This program helps to create a to-do list.===
+
+Usable commands:
+
+list: 'Shows current to-dos',
+add: 'Adds a to-do item. All the words behind add are entered as 1 to-do item to the list.'
+remove: 'Removes a to-do item by its 1-base index.'
+reset: 'Removes all to-do items from the list.'
+update: 'Updates a to-do item with new text.'
```

Comment:
You can write these command this way to make it more helpful for the end user:
`update [number] [todo]: Updates todo number [number] in the list with [todo]`
As a user when I check the help file I want to know how to use that command :)

Reply:
Help list has been improved